app/cafe.py

```diff
@@ -0,0 +1,23 @@
+import datetime
+from typing import Union
+from app.errors import (
+    NotWearingMaskError,
+    OutdatedVaccineError,
+    NotVaccinatedError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> Union[str, None]:
+        if not visitor.get("vaccine"):
+            raise NotVaccinatedError("The visitor is not vaccinated")
+        if visitor.get("vaccine", {}).get("expiration_date") is not None\
+                and datetime.date.today() >\
+                visitor.get("vaccine", {}).get("expiration_date"):
```

Comment:
```
if (visitor.get("vaccine", {}).get("expiration_date") is not None)
    and datetime.date.today() >
    visitor.get("vaccine", {}).get("expiration_date")):
```

Reply:
**app/cafe.py:18:43: W504 line break after binary operator**, to avoid this mistake i will put the ">" symbol on 19th line